### ISSUE TITLE
Magisk Canary 18114 automatically boots in fastboot

### ISSUE THREAD

**Issue by KaMyKaSii** [topjohnwu/Magisk#1191]
After installing Canary 18114, my Zenfone 4 automatically boots in fastboot. I tried a clean install but with the same result. The gdrive link below contains the stock boot.img, the patched boot.img with Canary 18114 and the asdf partition dump (which contains the last kernel logs and other logs)

https://drive.google.com/file/d/1_HTmTh258ErvpT8hRnRDfP552xiLReFa/view?usp=drivesdk

**Comment by topjohnwu** [comment]
When did it last worked? Or it has never worked?
@shakalaca do you have any input on this?

**Comment by KaMyKaSii** [comment]
@topjohnwu the last functional version I know is the 18112. I did not get to test 18113 because when I went to upgrade Magisk, the 18114 build was already available. But I can test it and report the result (when I wake up, I'm going to bed now). But I must tell you that since the fix in my last issue (#1086) I need around three boot attempts to finally get Android boot, the other two attempts always get stuck in the boot logo for around thirty seconds and then enter fastboot mode
